### DISCUSSION TITLE
Is there a way to traduce "Week" on Week number

### DISCUSSION THREAD

**Discussion by Nemesis24** [totaldebug/atomic-calendar-revive#1303] [Support]
hi,

Is there a way to traduce "Week" on Week number or only show week number?

**Comment by marksie1988** [comment]
traduce?
  - speak badly of or tell lies about

Sorry im not sure what it is you would like to do

**Comment by Nemesis24** [comment]
ok when you show week number that show "Week 02" in french (yes i know sorry for my poor english) "Semaine 02".

**Reply by marksie1988** [reply]
Ah ok, this should happen automatically but in some cases it doesn't work. I'm yet to find a good solution to resolve this issue.

**Reply by marksie1988** [reply]
This should be fixed now @Nemesis24, there was a bug in the translation code that caused this to happen. Please let me know if the latest version resolved this for  you
